### DISCUSSION TITLE
Unable to verify document. GitHub problem

### DISCUSSION THREAD

**Discussion by thesumedh** [community/community#72585] [GitHub Education]
Hi, Im college student and i have applied for the github student pack with my college fee receipt.
 It was rejected saying "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted.
Additionally my college doesn't provide .edu mail. 
It also saying to add billing information. 
Does college students have credit card? If I had Money why I would apply for student benifits 😔.

**Comment by kidpariz77** [comment]
yes

**Comment by ArnavVarshney** [comment]
> It also saying to add billing information. Does college students have credit card? If I had Money why I would apply for student benifits 😔.

You only have to add your billing information and not necessarily a payment method.
